Data/Ascii/AsciiHeaderIO.cpp

```diff
@@ -0,0 +1,140 @@
+// This file is part of the ComPWA framework, check
+// Copyright (c) 2013 The ComPWA Team.
+// https://github.com/ComPWA/ComPWA/license.txt for details.
+
+#include <algorithm>
+#include <fstream>
+#include <regex>
+#include <sstream>
+
+#include "Core/Exceptions.hpp"
+#include "Data/Ascii/AsciiHeaderIO.hpp"
+
+namespace ComPWA {
+namespace Data {
+namespace Ascii {
+
+/// @cond INTERNAL
+
+namespace std_fix {
+std::string tolower(std::string s) {
+  std::transform(s.begin(), s.end(), s.begin(),
+                 [](unsigned char c) { return std::tolower(c); });
+  return s;
+}
+} // namespace std_fix
+
+/// Check if a line starts with a float/int value
+bool isValueLine(const std::string &line) {
+  std::stringstream ss(line);
+  float value;
+  return (bool)(ss >> value);
+}
+
+/// @endcond
+
+std::string AsciiHeader::extractHeaderContent(std::istream &InputStream) {
+  /// -# Define regular expressions
+  static const std::regex HeaderBegin("<header>", std::regex_constants::icase);
```

Comment:
I thought its yaml style? So, `Header:`. Why you support <header>?

Reply:
It was decided in the last meeting that we stack the yaml between open and closing tags (html/xml) in this case. The file in itself is not yaml anyway because of the data outside the header section.

Reply:
But some tests use `Header:`, so that is also supported?

Reply:
Ah no I missed that. They all have been changed. Ok fine with me

Reply:
No, `Header` is not supported as begin/end tag, so the test files were modified as well, but I noticed a mistake in the doxygen string of AsciiDataIO.hpp now...